.github/workflows/on-main-branch-commit-push-liminal-node-to-ecr.yml

```diff
@@ -0,0 +1,83 @@
+---
+name: Builds and pushes aleph-node liminal to ECR on push to main branch
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build-liminal-node:
+    name: Build liminal node
+    uses: ./.github/workflows/_build-liminal-node.yml
+    secrets: inherit
+
+  push-liminal-node-image-to-ecr:
+    name: Push liminal node docker image to the ECR
+    needs: [build-liminal-node]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout aleph-node sources
+        uses: actions/checkout@v3
+
+      - name: Call action get-ref-properties
+        id: get-ref-properties
+        uses: ./.github/actions/get-ref-properties
+
+      - name: Download node production binary from artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: aleph-node-liminal
+          path: target/release/
+
+      - name: Build production docker node image
+        id: build-image
+        run: |
+          chmod +x target/release/aleph-node
+          docker build --tag aleph-node:latest -f ./docker/Dockerfile .
+
+      - name: Configure AWS Mainnet credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_MAINNET_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_MAINNET_SECRET_ACCESS_KEY }}
+          aws-region: eu-central-1
+
+      - name: Login to Public Amazon ECR
+        id: login-public-ecr
+        uses: docker/login-action@v2
+        with:
+          registry: public.ecr.aws
+          username: ${{ secrets.AWS_MAINNET_ACCESS_KEY_ID }}
+          password: ${{ secrets.AWS_MAINNET_SECRET_ACCESS_KEY }}
+        env:
+          AWS_REGION: us-east-1
+
+      - name: Push aleph-node image to ECR
+        env:
+          CURRENT_IMAGE: aleph-node-liminal
+          ECR_PUSH_IMAGE: public.ecr.aws/p6e8q1z1/aleph-node-liminal
+        run: |
+          docker tag ${{ env.CURRENT_IMAGE }} \
```

Comment:
Don't you need the `:latest` here as well?

Reply:
Or is it implicit?

Reply:
I would push both: ref and latest